src/classes/metadata/pdf.vala

```diff
@@ -229,7 +229,7 @@ namespace pdfpc.Metadata {
                     GLib.FileUtils.set_contents(this.pdfpc_fname, contents);
```

Comment:
It's an `||`, not a `&&`. pdfpc attempts to write the metadata file if either there is metadata to be written or an old metadata file already exists.